lib/sequenceserver/blast/report.rb

```diff
@@ -87,7 +87,8 @@ def extract_program_info(ir)
       def extract_params(ir)
         params = Hash[
           *ir[7].first.map { |k, v| [k.gsub('Parameters_', ''), v] }.flatten
-        ].merge(job.advanced_params)
+        ]
+        params = params.merge(job.advanced_params) unless job.advanced_params.nil?
```

Comment:
The reason why this is `nil` is that `job` is being loaded through a YAML file. It would be better to pass an actual instance of `Job` that way `parse_advanced` will get called and create the empty hash.